client/src/components/itemList/ItemCard.js

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import { ShowItemIcon } from './Icons';
+import PropTypes from 'prop-types';
+
+export const Card = styled.div`
+  width: 100%;
+  height: 45px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  font-size: 1.1rem;
+  border-bottom: 1px solid #cbcbcb;
+`;
+
+function ItemCard({ name, index, icon }) {
+  return (
+    <Card>
+      <div>{index}</div>
+      <div>{name}</div>
+      <div>{icon}</div>
+      <ShowItemIcon />
```

Comment:
```suggestion
   <div><ShowItemIcon /></div>
```

Maybe the SVG can also be wrapped by a `div` so you can control styles easier from the parent element. Please check my other comment for this.